### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Gustavonobreza**
Just a little fix, to better visualization.
😁

### CODE REVIEW

**Comment by jlooper** [comment]
Hi, I don’t think the change adds to readability so am closing

**Review by SmashedFrenzy16 — COMMENTED** [review]
ok

File: 7-bank-project/1-template-route/README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ In this lesson, we're going to lay out the foundations to create bank web app, u
 
 ### Prerequisite
 
-You need a local web server to test the web app we'll build in this lesson. If don't have one, you can install [Node.js](https://nodejs.org) and use the command `npx lite-server` from your project folder. It will create a local web server and open your app in a browser.
```
Comment (EmmStewie2709) [inline 7-bank-project/1-template-route/README.md:15]: D I R E T S O  •̀.̫•́

**Review by EmmStewie2709 — COMMENTED** [review]
D I R E T S O •̀.̫•́

**Pull request closed without merging**